bundles/org.openhab.binding.openthermgateway/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.openthermgateway-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-openthermgateway" description="OpenThermGateway Binding"
+		version="${project.version}">
```

Comment:
This needs to be on the same line as the feature tag.

Reply:
Adjusted